### PR TITLE
Test gateway creation

### PR DESCRIPTION
**Pull request by natalian98**
This PR adds a simple test to check if the gateway gets created, in attempt to help debugging issue [#94](https://github.com/canonical/istio-operators/issues/94).
The aim of this PR is to determine whether the gateway gets created in CI while no other charms are deployed and exclude a possible cause of other charms interfering istio-pilot in handling the config-changed event.

### CODE REVIEW

File: tests/test_bundle.py [hunk 1]
```diff
@@ -46,7 +46,10 @@ async def test_deploy_istio_charms(ops_test: OpsTest):
     istio_charms = await ops_test.build_charms(f"{charms_path}-gateway", f"{charms_path}-pilot")
 
     await ops_test.model.deploy(
-        istio_charms['istio-pilot'], application_name='istio-pilot', trust=True
+        istio_charms['istio-pilot'],
+        application_name='istio-pilot',
+        config={'default-gateway': 'kubeflow-gateway'},
```
Comment (DnPlas) [inline tests/test_bundle.py:51]: The name could be anything really, shall we change it to "test-gateway"?

**Review by DnPlas — COMMENTED** [review]
Thanks @natalian98 , let's rebase your branch to see the CI working on top of HEAD+your commits. I just left one comment, other than that, this LG.

File: tests/test_bundle.py (hunk 1 above)
Comment (natalian98) [reply tests/test_bundle.py]: Thanks for checking this! I rebased my branch and renamed the gateway.

**Review by DnPlas — APPROVED** [review]
LGTM, I still see some commits from other people in your branch. We can merge after you have cleaned that up. Thanks @natalian98

**Pull request closed without merging**